packages/react-native-session-replay/README.md

```diff
@@ -1,3 +1,54 @@
 # Session Replay for React Native
 
-Session Replay for React Native is not available yet.
+Mobile Session Replay expands visibility into your mobile applications by visually replaying each user interaction, such as taps, swipes, and scrolls. Visually replaying user interactions on your applications makes it easier to reproduce crashes and errors, as well as understand the user journey for making UI improvements.
+
+## Setup
+
+**Note:** Make sure you’ve setup and initialized the [Datadog React Native SDK][1] with views instrumentation enabled.
+
+To install with NPM, run:
+
+```sh
+npm install @datadog/mobile-react-native-session-replay
+```
+
+To install with Yarn, run:
+
+```sh
+yarn add @datadog/mobile-react-native-session-replay
+```
+
+## Enable Session Replay
+
+To enable Session Replay, import and call the `enable` method with your configuration. Below is an example setup:
+
+```js
+import { SessionReplay } from "@datadog/mobile-react-native-session-replay";
+
+SessionReplay.enable({
+    replaySampleRate: sampleRate // The percentage of sampled replays, in the range 0.0 - 100.0 (Default: 100.0).
+    textAndInputPrivacyLevel: TextAndInputPrivacyLevel.MASK_ALL, // Defines the way text and input (e.g text fields, checkboxes) should be masked.
```

Comment:
We have the default values here for sampling rate and image privacy, but maybe should add also for text and touch privacy